lib/model/model.go

```diff
@@ -1050,8 +1036,9 @@ func (m *model) ClusterConfig(deviceID protocol.DeviceID, cm protocol.ClusterCon
 		}
 	}
 
-	var paused []string
-	indexSenders := make(map[string]*indexSender, len(cm.Folders))
+	resendClusterConfig := false
```

Comment:
This is nasty. I don't think it's reasonable to have *our* internal state when we resume a folder depend on the counterpart noticing the change in our pause status and repeating an identical cluster config. I mean, can you even document that requirement in the BEP spec with a straight face? 😁